### DISCUSSION TITLE
球为什么不是女生呢

### DISCUSSION THREAD

**Discussion by This-is-XiaoDeng** [ITCraftDevelopmentTeam/Forum#5] [General]
我一直在想一个问题 @PowerAngelXD 为什么不是女生呢？

如果 @PowerAngelXD 是女生的话，应该很漂亮吧，女生会C++多帅

**Comment by This-is-XiaoDeng** [comment]
[doge]

**Comment by PowerAngelXD** [comment]
四个字：wcnm

**Reply by This-is-XiaoDeng** [reply]
（

**Reply by LzzKill** [reply]
正确的，客观的，合理的，中肯的，明晰的，真实的，辩证的，理性的，审慎的，深刻的，通达的，优美的，巧妙的，精辟的，雅正的，机智的，全面的，明白晓畅的，不偏不倚的，恰如其分的，滴水不漏的，不容质疑的，切中要害的，一针见血的，淋漓尽致的，深谙事理的，真知灼见的，发蒙振聩的，微言大义的，金声玉振的，透过现象看本质的，知其然而知其所以然的，可供世人仿效的，千古颠扑不破的

**Comment by YifanZhuZhu** [comment]
666

**Comment by zhao6888** [comment]
因为他有**DICK**
